widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -735,7 +735,7 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
                     dispatchImageCapture()
                 } else {
                     Utils.getActivity(context)
-                        .requestPermissions(
+                        ?.requestPermissions(
```

Comment:
Not critical: maybe it will be better to put `?.requestPermissions(` to the same line with `Utils.getActivity(context)`?